helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -223,6 +224,10 @@ charEncodingUtf8: false
 # sourcetype of each kind of log can be defined using the `sourcetype` field.
 # If `sourcetype` is not defined, `name` will be used.
 #
+# = limitRecentlyModified =
+# Limits the watching files that the modification time is within the specified time range when using * in path.
+# If `limitRecentlyModified` is not defined, this option will be disabled.
```

Comment:
It would be more clear if we explain it with an example. I.e. what would happen if we provide 24h.